tools/cli/commands/docker.js

```diff
@@ -713,8 +727,8 @@ export function dockerDefine( yargs ) {
 					handler: argv => execDockerCmdHandler( argv ),
 				} )
 				.command( {
-					command: 'phpunit-multisite',
-					alias: 'phpunit:multisite',
+					command: 'phpunit-jp-multisite',
+					alias: 'phpunit:jp:multisite',
```

Comment:
Given this alias has never existed, perhaps we don't need it?

Reply:
🤷